benchmark/coinmarketcap/coinmarketcap.com.bench.js

```diff
@@ -0,0 +1,100 @@
+import assert from 'assert';
+import { fox } from '../../src/index.js';
+import { objectCartesian, benchmarkOptions } from '../lib.js';
+import { matchesPatterns, runCrawlBenchmark } from '../lib/crawl.js'
+
+const coinmarketEval = {
```

Comment:
this and `coinmarketOptions` shouldn't be pulled out separately, it makes it harder to scan the test file this way